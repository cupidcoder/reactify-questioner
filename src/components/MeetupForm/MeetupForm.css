.MeetupFormContainer {
  margin-top: 6rem;
  margin-left: 15%;
  width: 70%;
}

.MeetupFormContainer h2 {
  text-align: center;
  color: #305b8c;
  padding: 1rem;
  font-weight: 700;
}

.MeetupForm {
  width: 50%;
  margin: 0 auto;
  box-shadow: 6px 3px 11px 3px rgba(128,128,128,0.61);
  border-radius: 3px;
}

.Input input {
  display: block;
  width: 60%;
  margin: 0 auto;
  padding: 0.7rem;
  margin-bottom: 1rem;
}


.Button {
  width: 8.4rem;
  margin: 0 auto;
  padding-bottom: 1rem;
}
