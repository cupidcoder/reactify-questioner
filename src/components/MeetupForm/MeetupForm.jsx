import React, { Component } from 'react';
import { connect } from 'react-redux';
import Button from '../Button/Button';
import { incrementAction } from '../../actions/testAction';
import style from './MeetupForm.css';

class MeetupForm extends Component {
  render() {
    const { increase } = this.props;
    return (
      <div className={style.MeetupFormContainer}>
        <div className={style.MeetupForm}>
          <h2>Create Meetup</h2>
          <form>
            <div className={style.Input}>
              <input type='text' id='topic' placeholder='Topic' />
            </div>
            <div className={style.Input}>
              <input type='text' id='description' placeholder='Description' />
            </div>
            <div className={style.Input}>
              <input type='text' id='location' placeholder='Location' />
            </div>
            <div className={style.Input}>
              <input type='text' id='date' placeholder='Date' />
            </div>
            <div className={style.Button}>
              <Button onClick={() => increase()}>Create</Button>
            </div>
          </form>
        </div>
      </div>
    );
  }
}

const mapDispatchToProps = dispatch => ({
  increase: () => dispatch(incrementAction())
});

export default connect(null, mapDispatchToProps)(MeetupForm);
