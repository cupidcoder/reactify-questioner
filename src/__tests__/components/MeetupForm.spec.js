import React from 'react';
import { shallow } from 'enzyme';
import MeetupForm from '../../components/MeetupForm/MeetupForm';

describe('MeetupForm', () => {
  it('Should correctly save snapshot of the MeetupForm', () => {
    const wrapper = shallow(<MeetupForm />);
    expect(wrapper).toMatchSnapshot();
  });
});
