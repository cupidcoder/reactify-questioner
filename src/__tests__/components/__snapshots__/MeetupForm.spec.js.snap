// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetupForm Should correctly save snapshot of the MeetupForm 1`] = `ShallowWrapper {}`;
